packages/web-api/src/WebClient.ts

```diff
@@ -220,11 +216,10 @@ export class WebClient extends Methods {
 
   /**
    * Generic method for calling a Web API method
-   *
    * @param method - the Web API method to call {@link https://api.slack.com/methods}
    * @param options - options
    */
-  public async apiCall(method: string, options: WebAPICallOptions = {}): Promise<WebAPICallResult> {
```

Comment:
Any areas where we still want things to extend from a shapeless foundation, we can instead use `Record<string, unknown>` or `Record<string, any>` to communicate "an object with any shape but with root keys being strings".

Reply:
yeah, i don't see any issues with this change